Library/Homebrew/official_taps.rb

```diff
@@ -8,6 +8,14 @@
   tex
 ].freeze
 
+OFFICIAL_CASK_TAPS = %w[
```

Comment:
> There may also be a way

I thought this implied that you know of another way to do this, so I haven't looked into it.